schema/test/unit/computed_columns/form_change_calculated_total_payment_amount_to_date_test.sql

```diff
@@ -0,0 +1,140 @@
+
+begin;
+
+select plan(4);
+
+insert into cif.project_revision(id, change_status)
+  overriding system value
+  values
+  (1, 'pending'),
+  (2, 'pending'),
+  (3, 'pending'),
+  (4, 'pending');
+
+insert into cif.form_change(
+  id,
+  operation,
+  form_data_schema_name,
+  form_data_table_name,
+  form_data_record_id,
+  project_revision_id,
+  json_schema_name,
+  new_form_data
+) overriding system value
+-- funding parameter form changes
+values (
+  1,
+  'create',
+  'cif',
+  'funding_parameter',
+  1,
+  1,
+  'funding_parameter_IA',
+  '{"projectId": 1,  "maxFundingAmount": 100}'
+),
+(
+  2,
+  'create',
+  'cif',
+  'funding_parameter',
+  2,
+  2,
+  'funding_parameter_IA',
+  '{"projectId": 1,  "maxFundingAmount": 100}'
+),
+(
+  3,
+  'update',
+  'cif',
+  'funding_parameter',
+  3,
+  3,
+  'funding_parameter_IA',
+  '{}'
+),
+(
+  4,
+  'create',
+  'cif',
+  'funding_parameter',
+  4,
+  4,
+  'funding_parameter_IA',
+  '{"projectId": 1,  "maxFundingAmount": 100}'
+),
+(
+  5,
+  'create',
+  'cif',
+  'project_summary_report',
+  5,
+  2,
+  'project_summary_report',
+  '{"projectSummaryReportPayment":20}'
+),
+(
+  6,
+  'create',
+  'cif',
+  'project_summary_report',
+  6,
+  4,
+  'project_summary_report',
+  '{"projectSummaryReportPayment":20}'
+),
+(
+  7,
+  'create',
+  'cif',
+  'project_summary_report',
+  7,
+  4,
+  'project_summary_report',
+  '{"projectSummaryReportPayment":25}'
+),
+(
+  8,
+  'create',
+  'cif',
+  'project_summary_report',
+  8,
+  4,
+  'project_summary_report',
+  '{"projectSummaryReportPayment":30}'
+);
+
+select is(
```

Comment:
I'd also add tests for:
- when the summary payment is filled in but the max funding isn't
- when summary payment and/or max funding is archived